ote/src/cljs/ote/views/admin/monitor.cljs

```diff
@@ -1,9 +1,10 @@
-(ns ote.views.monitor
+(ns ote.views.admin.monitor
   (:require [reagent.core :as r]
             [ote.ui.common :as ui-common]
             [ote.localization :refer [tr tr-key]]
-            [ote.app.controller.monitor :as controller]
-            [cljsjs.chartjs]))
+            [ote.app.controller.monitor :as monitor-controller]
```

Comment:
While this is a matter of convention and taste, 
I'd prefer shorter `:as` aliases to save a bit of space in the source code file. 
We don't have an agreed convention, but a common one is to use prefix/suffixes like 
controller=>ctrl, view=>vw.

Reply:
I'm not happy with this name also. But I don't like short names that don't tell what they are. ctrl-monitor would be better name, but then we need to change all the names we have been using. :)

Reply:
Yes. "Find & Replace > replace all" hides the pain ;)